nibabel/tests/test_arrayproxy.py

```diff
@@ -327,3 +332,188 @@ def check_mmap(hdr, offset, proxy_class,
             # Check invalid values raise error
             assert_raises(ValueError, proxy_class, fname, hdr, mmap='rw')
             assert_raises(ValueError, proxy_class, fname, hdr, mmap='r+')
+
+
+# An image opener class which counts how many instances of itself have been
+# created
+class CountingImageOpener(ImageOpener):
+
+    numOpeners = 0
```

Comment:
I believe this should be `num_openers` (mixedCase not allowed unless it's already the default naming scheme : https://www.python.org/dev/peps/pep-0008/

Reply:
A throw-back from many years as a java programmer :)